### PR TITLE
Data generation script

### PR DESCRIPTION
**Pull request by michaelrtobias**

### CODE REVIEW

**Comment by michaelrtobias** [comment]
@bland0 @chanceknguyen

File: db/dataGeneration.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+      const genre2 = listPicker(genreList);
+      const genre3 = listPicker(genreList);
+      const developer = capitalize(faker.random.word());
+      const publisher = capitalize(faker.random.word());
+      const releaseDate = faker.date.between('2000-01-01', '2020-11-13');
+      const steamAcheivment1 = 'https://s3.us-east-2.amazonaws.com/images.for.hrr/869-100x100.jpg';
+      const steamAcheivment2 = 'https://s3.us-east-2.amazonaws.com/images.for.hrr/818-100x100.jpg';
+      const steamAcheivment3 = 'https://s3.us-east-2.amazonaws.com/images.for.hrr/612-100x100.jpg';
+
+      const languageName1 = listPicker(languageList);
```
Comment (bland0) [inline db/dataGeneration.js:50]: this seems like something you could loop over

File: db/modelsPostgres/Game.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+		memory: '2 GB RAM',
+		graphics: '512 Mb',
+		directX: 'Version 9.0',
+		storage: '900 MB available space',
+		},
+	genre: ['cats', 'adventure', 'hello', 'kitty', 'racing'],
+	developer: 'bethesda',
+	publisher: 'bioware',
+	releaseDate: 'hopefully never',
+	steamAcheivments:['https://s3.us-east-2.amazonaws.com/images.for.hrr/869-100x100.jpg', 'https://s3.us-east-2.amazonaws.com/images.for.hrr/818-100x100.jpg','https://s3.us-east-2.amazonaws.com/images.for.hrr/612-100x100.jpg'],
```
Comment (bland0) [inline db/modelsPostgres/Game.js:39]: if s3.us-east-2.amazonaws.com/images.for.hrr/ will always be present in this URL, then consider only storing the file name

File: server/index.js [hunk 3]
```diff
@@ -12,7 +12,10 @@ app.use(express.static(path.join(__dirname, '../client/build')))
 
 //db switcher
 //mongoDB
-const db = require('../db/mongoDBFunctions.js')
+const db = require('../db/mongoDBFunctions.js');
+
+//postgresQL
+// const db = require('../db/indexPostgreSQL.js');
 
 app.get('/moist-air/', db.getAllGames);
```
Comment (bland0) [inline server/index.js:20]: what is at this root endpoint?

**Review by bland0 — APPROVED** [review]
Only major thing is the repeated stuff in the generation script.

File: server/index.js (hunk 3 above)
Comment (michaelrtobias) [reply server/index.js:20]: it is all of the games in the db. Probably will not be necessary now with all these records

**Pull request merged**